test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -97,11 +98,21 @@ def run_ec2_tensorflow_inference(image_uri, ec2_connection, ec2_instance_ami, gr
 
     docker_cmd = "nvidia-docker" if "gpu" in image_uri else "docker"
     if is_neuron:
+        #For 2.5 using rest api port instead of grpc since using curl for prediction instead of grpc
+        if str(framework_version).startswith(TENSORFLOW2_VERSION):
+            model_name= "simple"
+            model_path = os.path.join(serving_folder_path, "models", model_name)
+            src_port = "8501"
+            dst_port = "8501"
+        else:
+            src_port = grpc_port
+            dst_port = "8500"
```

Comment:
Is the indentation correct here? Should the `else` condition apply to the outer if-condition?

Reply:
Yes. it is correct.. #For 2.5 using rest api port instead of grpc since using curl for prediction instead of grpc.